README.md

```diff
@@ -118,6 +120,63 @@ val rabbitMQProducer: Resource[Task, RabbitMQProducer[Task, Bytes]] = {
   }
 ```
 
+#### Streaming support
+
+It seems quite natural to process RabbitMQ queue with a streaming app.
+[`StreamingRabbitMQConsumer`](core/src/main/scala/com/avast/clients/rabbitmq/StreamingRabbitMQConsumer.scala) provides you an
+[`fs2.Stream`](https://fs2.io/) through which you can easily process incoming messages in a streaming way.
+
+Notice: Using this functionality requires you to know some basics of [FS2](https://fs2.io/guide.html#overview) library. Please see it's official
+guide if you're not familiar with it first.
+
+```scala
+// skipping imports and common things, they are the same as in general example above
+
+val consumerConfig = StreamingConsumerConfig( // notice: StreamingConsumerConfig vs. ConsumerConfig
```

Comment:
How is `StreamingConsumerConfig` different from `ConsumerConfig`. Ultimately they should be the same because the only thing that differs is the "output" type - callback function or `fs2.Stream`. Right?

Reply:
There **are** few differences - one valu added, two removed. It seemed to me as a good idea to have them split from the beginning but now they are indeed not equal.